examples/inverse/plot_lcmv_beamformer_volume.py

```diff
@@ -61,10 +61,16 @@
 data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
                                   method='shrunk')
```

Comment:
I get warnings when running this example. This should not happen:

```
/Users/alex/work/src/mne-python/examples/inverse/plot_lcmv_beamformer_volume.py:73: DeprecationWarning: max_ori_out and abs is deprecated and will be removed in 0.16. Set it to signed to remove this warning, this will return signed time series.
  pick_ori='max-power', weight_norm='nai')
max_ori_out and abs is deprecated and will be removed in 0.16. Set it to signed to remove this warning, this will return signed time series.
```